pkg/proof/querier.go

```diff
@@ -156,3 +162,10 @@ func ParseNamespace(rawShares []shares.Share, startShare, endShare int) (appns.N
 	}
 	return startShareNs, nil
 }
+
+func safeConvert(x int64) (int, error) {
+	if math.MinInt <= x && x <= math.MaxInt {
+		return int(x), nil
+	}
+	return 0, fmt.Errorf("invalid int: %d", x)
```

Comment:
[optional]
guess it's better to stick to the same error message in the other function. Alternatively, we could create an error type for it.

```suggestion
	return 0, fmt.Errorf("value %d is too large to convert to int", x)
```

Reply:
Agreed! Thanks